ls1intum/Artemis
# `Programming exercises`: Switch tutorial to IntelliJ

### Checklist
#### General


#### Client

### Motivation and Context


Artemis is switching from Maven to Gradle and from SourceTree+Eclipse to IntelliJ as default choices. So the tutorial should follow along.

### Description

This PR updates some of the steps and all included videos of the interactive tutorial

### Steps for Testing

Deploy this PR to test server 10 and head to the tutorial course: https://artemis-test10.artemis.in.tum.de/courses/2/exercises

Prerequisites:
- 1 Instructor (e.g. artemis_test_user_11)
- TS10 with tutorial course

1. Log in to Artemis
2. Navigate to the tutorial course
3. Restart the tutorial on the top right by clicking on the login
4. Follow along with the tutorial and check that the steps are coherent

### Review Progress






#### Code Review
#### Manual Tests
